alg_test.go

```diff
@@ -5,9 +5,9 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/golang-jwt/jwt/v4"
+	"github.com/golang-jwt/jwt"
 
-	"github.com/MicahParks/keyfunc"
+	keyfunc "github.com/pmishchenko-ua/jwt-keyfunc"
```

Comment:
oh man... this is very risky - how to guarantee you update all these imports before landing?

Reply:
I will update them after landing - links in this repo are in README, examples and tests, so not a big risk.